### PR TITLE
Development for review

### PR DESCRIPTION
**Pull request by shloch**

### CODE REVIEW

File: src/contact.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+let pre_txt = `<h1>Contact Us</h1>
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation</p>
+<h4>Tel : 456-908-890</h4>
+<h4>Email : restaurant@microverse.code</h4>
+<h4>Location : STandford Road , SILICON VALLEY</h4>
+<h4>OPENING HOURS : 10am - 7pm</h4>`;
+
+function contact_txt() {
+    var element = document.createElement("div");
```
Comment (SunnyAsar) [inline src/contact.js:9]: we really don't need comments in our code. lets take them out app wide.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+home_txt();
+//-----------------------------------------
+const home_link = document.querySelector("#home_link");
+home_link.addEventListener("click", e => {
+    home_txt();
+    e.preventDefault();
+});
+
+const about_link = document.querySelector("#about_link");
+about_link.addEventListener("click", e => {
```
Comment (SunnyAsar) [inline src/index.js]: But isn't this repetition? Here is my suggestion:
- let the tab buttons all have the same click handler and pass something we can identify them by. let have our imported modules in an array, so depending on what is passed we render a different tab. that way we don't repeat.

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
Hey @shloch 😃 ,
your app is super fine! and the code is clean too. too bad i couldn't approve it immediately which i wanted to. but here are the problems.
- let's remove the comments
- we need to have linting setup on the repository and have a linting config in our code.
https://github.com/microverseinc/linters-config/tree/master/javascript
- Git flow - Changing this now is a lot of work but let's use the correct git flow next time https://gitlab.com/microverse/guides/tse/how_to_be_a_tse/blob/master/code_review_checklist/03_flows.md

#### Finally
- I made some suggestions about how you change tabs
please do this quick fix and let's move on. Cheers!

File: src/index.js (hunk 2 above)
Comment (shloch) [reply src/index.js]: @SunnyAsar  Thanks for the suggestion, i had brought some amelioration to the code

File: src/contact.js (hunk 1 above)
Comment (dipto0321) [inline src/contact.js:9]: Let's use `const` not `var`

File: src/home.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+let home_pre_txt = `<h1 class="anim">WELCOME TO SHLOCH RESTAURANT</h1>
+<p>
+    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor
+    in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+</p>`;
+
+function home_txt() {
+    var element = document.createElement("div");
```
Comment (dipto0321) [inline src/home.js:8]: Use `const`

File: src/menu.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        <h4>Lorem ipsum dolor sit amet</h4>
+    </div>
+
+<div class="col-6 first-col">
+    <img src="assets/image1.jpeg" class="img-fluid shadow-lg p-3 mb-5 bg-white rounded responsive"> 
+</div>
+`;
+
+function menu_txt() {
+    var element = document.createElement("div");
```
Comment (dipto0321) [inline src/menu.js:15]: Use `const`

File: src/navbar.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        <a class="nav-item nav-link" href="#home">HOME</a>
+        <a class="nav-item nav-link" href="#menu">MENU</a>
+        <a class="nav-item nav-link" href="#about">ABOUT</a>
+        <a class="nav-item nav-link" href="#contact">CONTACT</a>
+    </div>
+</div>
+</nav>`;
+
+function nav_menu() {
+    var nav_menu = document.createElement("div");
```
Comment (dipto0321) [inline src/navbar.js:19]: Use `const`

File: src/navbar.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    nav_menu.setAttribute("id", "header");
+    nav_menu.innerHTML = nav_pre_txt;
+
+    var BODY = document.querySelector("body");
```
Comment (dipto0321) [inline src/navbar.js:23]: Use `const`

File: src/index.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+import { contact_txt } from "./contact";
+import { home_txt } from "./home";
+
+nav_menu();
+home_txt();
+
+const tabs = document.querySelectorAll(".navbar-nav a");
+tabs.forEach((tab, index) => {
+    tab.addEventListener("click", event => {
+        console.log(event.currentTarget.hash.slice(1));
```
Comment (dipto0321) [inline src/index.js:15]: Please remove this console

**Review by dipto0321 — APPROVED** [review]
Hi @shloch ,

Good job 👏 . Please always use eslint for your local like `eslint --init` and then let it create its own config file. Then you have to rewrite stickler 
```yaml
linters:
  eslint:
    config: "./.eslintrc.js"
```
For this project I noticed stickler is not working. So before merging this project please add eslint the regular way with `airbnb` style guide and fix all the issues. This is from me in advance 👍 

Thanks 🙏

**Pull request merged**